app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -76,5 +84,11 @@ public function panel(Panel $panel): Panel
             ->authMiddleware([
                 Authenticate::class,
             ]);
+
+        if (config('panel.filament.top-navigation') === false) {
+            $panel->renderHook('panels::sidebar.footer', fn () => view('filament.footer'));
+        } else {
+            //$panel->renderHook('panels::topbar.start', fn () => view('filament.footer')); // TODO find a fix for the footer if topbar is enabled
+        }
```

Comment:
I don't actually see the footer before or after my changes.

Reply:
Thats weird. I do see it as you can see on the screenshot? Do you have topbar on? Because then it does not work only with sidebar for now

Reply:
Fixed it. For some reason this bit of code did not work anymore, but it did work when i last tested it and send my screenshots so no idea why it does not now.